discord_bots/utils.py

```diff
@@ -825,3 +829,174 @@ def get_team_name_diff(
+        if interaction:
+            message = interaction.message
+            guild = interaction.guild
+        elif guild:
+            message = None
+        else:
+            raise Exception("No Interaction or Guild on _movegameplayers")
+
+        in_progress_game = (
+            session.query(InProgressGame)
+            .filter(InProgressGame.id.startswith(game_id))
+            .first()
+        )
+        if not in_progress_game:
+            if message:
+                await send_message(
+                    message.channel,
+                    embed_description=f"Could not find game: {game_id}",
+                    colour=Colour.red(),
+                )
+                return
+            return
+
+        team0_ipg_players: list[InProgressGamePlayer] = session.query(
+            InProgressGamePlayer
+        ).filter(
+            InProgressGamePlayer.in_progress_game_id == in_progress_game.id,
+            InProgressGamePlayer.team == 0,
+        )
+        team1_ipg_players: list[InProgressGamePlayer] = session.query(
+            InProgressGamePlayer
+        ).filter(
+            InProgressGamePlayer.in_progress_game_id == in_progress_game.id,
+            InProgressGamePlayer.team == 1,
+        )
+        team0_player_ids = set(map(lambda x: x.player_id, team0_ipg_players))
+        team1_player_ids = set(map(lambda x: x.player_id, team1_ipg_players))
+        team0_players: list[Player] = session.query(Player).filter(Player.id.in_(team0_player_ids))  # type: ignore
+        team1_players: list[Player] = session.query(Player).filter(Player.id.in_(team1_player_ids))  # type: ignore
+
+        be_voice_channel: VoiceChannel | None = None
+        ds_voice_channel: VoiceChannel | None = None
+        ipg_channels: list[InProgressGameChannel] | None = (
+            session.query(InProgressGameChannel)
+            .filter(InProgressGameChannel.in_progress_game_id == in_progress_game.id)
+            .all()
+        )
+        for ipg_channel in ipg_channels or []:
+            discord_channel: discord.abc.GuildChannel | None = guild.get_channel(
+                ipg_channel.channel_id
+            )
+            if isinstance(discord_channel, VoiceChannel):
+                # This is suboptimal solution but it's good enough for now. We should keep track of each team's VC in the database
+                if discord_channel.name == in_progress_game.team0_name:
+                    be_voice_channel = discord_channel
+                elif discord_channel.name == in_progress_game.team1_name:
+                    ds_voice_channel = discord_channel
+
+        coroutines = []
+        for player in team0_players:
+            if player.move_enabled and be_voice_channel:
+                member: Member | None = guild.get_member(player.id)
+                if member:
+                    member_voice: VoiceState | None = member.voice
+                    if member_voice:
+                        if member_voice.channel:
```

Comment:
nit: merge these two if statements to remove an indentation level: `if member_voice and member_voice.channel`